packages/wrangler-devtools/patches/0002-Show-fallback-image-on-Safari.patch

```diff
@@ -1778,15 +1778,12 @@ literal 0
 HcmV?d00001
 
 diff --git a/front_end/entrypoint_template.html b/front_end/entrypoint_template.html
-index 1edaec9604..945aa6d816 100644
+index 8e0e11ca35..ad217ee787 100644
 --- a/front_end/entrypoint_template.html
 +++ b/front_end/entrypoint_template.html
-@@ -14,7 +14,30 @@
-     }
-   }
+@@ -16,5 +16,28 @@
  </style>
--<meta http-equiv="Content-Security-Policy" content="object-src 'none'; script-src 'self' 'unsafe-eval' https://chrome-devtools-frontend.appspot.com">
-+<meta http-equiv="Content-Security-Policy" content="object-src 'none'; script-src 'sha256-7cSoIgLSn2W/Jf+ovCaVMDQtS5K1xhtFhl5WQtvvAaQ=' 'self' 'unsafe-eval' https://chrome-devtools-frontend.appspot.com">
+ <meta http-equiv="Content-Security-Policy" content="object-src 'none'; script-src 'self' https://chrome-devtools-frontend.appspot.com">
```

Comment:
Does this break the CSP, given the inline script we inject to this file?

Reply:
This seems to work locally (with the preview URL) - is there some way to test?

Reply:
When I open the preview URL, I get the following error:
`Refused to execute inline script because it violates the following Content Security Policy directive: "script-src 'self' https://chrome-devtools-frontend.appspot.com". Either the 'unsafe-inline' keyword, a hash ('sha256-nR3ouvAaRD5oAvx24d8aFSv1EH9XKi9uVvoOsvVuW8U='), or a nonce ('nonce-...') is required to enable inline execution.`